tv_apps/smart-home/js/home.js

```diff
@@ -784,13 +784,14 @@
     },
 
     onCardPickerHide: function() {
+      this.mode = '';
```

Comment:
We need to reset the `mode` to make elements in main page appear so that they can be focused correctly during `saveToNewFolder` and `updateFolder` running.